src/routes/userRouter.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from "express";
+import userControllers from "../modules/user/controller/userControllers";
+import { validateUserId, validateUserStatus, checkUserExists } from "../modules/user/validation/userValidations";
+
+const router: Router = Router()
+
+router.put("/update-user-status/:id", validateUserId, validateUserStatus, checkUserExists, userControllers.updateUserStatus);
```

Comment:
Please follow the code convention we are using in our project. Firstly, we are not using Joi Validations in this way.